kythe/typescript/indexer.ts

```diff
@@ -1549,12 +1565,31 @@ class Visitor {
    *     ((local var)|const|let) <name>: <type>( = <initializer>)?
    * where `(local var)` is the declaration of a variable in a catch clause.
    */
-  emitVariableDeclarationCode(decl: ts.VariableDeclaration, declVName: VName) {
+  emitVariableDeclarationCode(decl: ts.VariableDeclaration|ts.BindingElement, declVName: VName) {
```

Comment:
Is there a better name for this function?

Reply:
Ah I guess your other CL renames it...

Reply:
Yeah i'll defer to the other CL